layouts/_default/single.html

```diff
@@ -20,6 +20,23 @@ <h1 class="post-title"><a href="{{ .Permalink }}">{{ .Title | markdownify }}</a>
       <img src="{{ . | absURL }}" class="post-cover" />
     {{ end }}
 
+    {{ if .Params.Toc }}
+    <div class="table-of-contents">
+      {{ if .Params.Toc }}
```

Comment:
I dont think this is needed since you are already checking it at [line 23](https://github.com/panr/hugo-theme-terminal/pull/27/files#diff-092fccdb32952e423c5eb7c14f9be7c6R23)

Reply:
my bad, nice catch